src/MCPServer/lib/assets/workflow.json

```diff
@@ -7526,21 +7526,21 @@
                 "stdout_file": null
             },
             "description": {
-                "en": "Persistent identifier (PID) declaration"
+                "en": "Load persistent identifiers from metadata/identifiers.json"
```

Comment:
> Persistent identifier (PID) declaration

I feel like this warrants more discussion in this GitHub issue as you have done with the decision to move this to a different microservice group. This is the original title for the microservice we came up with for the client in `1.9` so it's good to make that open in there too. 

The docs are a little more ambiguous about the title of the service [here](https://www.archivematica.org/en/docs/archivematica-1.10/user-manual/transfer/transfer/#transfers-with-existing-persistent-identifiers). This PR does introduce a need to just fix those up so the new microservice group is reflected, and the new microservice name if we change it.

Reply:
My bad. I discussed the title change in Slack with @peterVG. I should have raised it in the Github issue. And yes, I was thinking of adding one of those tasks checkboxes to the PR description regarding the documentation change but I need to figure out how to do that.